.azure-pipelines.yml

```diff
@@ -269,6 +269,34 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-18.04'
```

Comment:
use `ubuntu-latest` (`ubuntu-18.04` is for building snap... because snap :trollface: )

Reply:
Isn't it bad if we generate wheels for a too high Python version? I'm not sure how this generation works under the hood, but aren't we supposed to launch bdist_wheel on a distribution with python 3.6?

Reply:
Parsec is currently pure-python (you can see this by the `none-any` part in the name of the wheel generated) so this is not a trouble.
Building binary wheel is much more complicated (see manylinux), we will have plenty of fun when we'll need this ;-)

Reply:
Fixed!